tests/javapc/src/main/scala/tests/pc/BaseJavaHoverSuite.scala

```diff
@@ -0,0 +1,58 @@
+package tests.pc
+
+import java.nio.file.Paths
+
+import scala.meta.internal.metals.CompilerOffsetParams
+import scala.meta.internal.metals.CompilerRangeParams
+import scala.meta.internal.mtags.MtagsEnrichments._
+
+import munit.Location
+import munit.TestOptions
+import tests.TestHovers
+import java.net.URI
+
+class BaseJavaHoverSuite extends BaseJavaPCSuite with TestHovers { // todo: General Logic
```

Comment:
```suggestion
class BaseJavaHoverSuite extends BaseJavaPCSuite with TestHovers {
```
that's probably done already?